src/main/java/io/cdap/plugin/gcp/bigquery/sink/AvroRecordWriter.java

```diff
@@ -103,6 +109,7 @@ private void createFileWriter(Schema writerSchema) throws IOException {
   public void close(TaskAttemptContext context) throws IOException {
     if (mAvroFileWriter != null) {
       mAvroFileWriter.close();
```

Comment:
Also, as `mAvroFileWriter` depends upon `outputStream` does it make sense to call `outputstream.close` in `mAvroFileWriter.close` instead of explicitly calling it here? I am assuming `outputstream` is used just for `mAvroFileWriter` correct me if this is wrong.

Reply:
Yes you are write `AvroFileWriter.close` internally closes `outputstream` . 
Thanks